DXMainClient/DXGUI/Multiplayer/GameLobby/GameLobbyBase.cs

```diff
@@ -1593,6 +1593,13 @@ private void WriteMap(PlayerHouseInfo[] houseInfos)
                 IniFile mpGlobalCodeIni = new IniFile(SafePath.CombineFilePath(ProgramConstants.GamePath, "INI", "Map Code", "MultiplayerGlobalCode.ini"));
                 MapCodeHelper.ApplyMapCode(mapIni, mpGlobalCodeIni);
             }
+            else
+            {
+                // Avoid writing the original filename to spawnmap.ini MP games, as it may vary between systems. 
```

Comment:
What do you mean by adding an avoid writing comment in front of the codes that writes this value